api/src/services/specifications.test.ts

```diff
@@ -1,26 +1,29 @@
 import knex, { Knex } from 'knex';
 import { getTracker, MockClient, Tracker } from 'knex-mock-client';
 import { CollectionsService, FieldsService, RelationsService, SpecificationService } from '../../src/services';
+import { describe, beforeAll, afterEach, it, expect, vi, beforeEach } from 'vitest';
 
-jest.mock('../../src/database/index', () => {
-	return { getDatabaseClient: jest.fn().mockReturnValue('postgres') };
+vi.mock('../../src/database/index', () => {
+	return { getDatabaseClient: vi.fn().mockReturnValue('postgres') };
 });
-jest.requireMock('../../src/database/index');
+vi.mock('../../src/database/index');
 
 class Client_PG extends MockClient {}
 
 describe('Integration Tests', () => {
-	let db: jest.Mocked<Knex>;
+	//TODO: replace with correct vitest api call
+	let db: vi.Mocked<Knex>;
```

Comment:
IMO, we can simply use `Knex` here:

```suggestion
	let db: Knex;
```